build.gradle

```diff
@@ -405,6 +406,19 @@ task integTestRemote(type: RestIntegTestTask) {
     }
 }
 
+
+// test retry configuration
```

Comment:
Can we add a TODO here for a long term fix, I'm just concerned that we'll miss catching some flaky issues that may help us identify some gaps/ improvements in the future. But as a short term fix, this works wonderfully.

Reply:
And perhaps with this change we can reset the `fail fast` setting back to its default within our CI workflow

Reply:
Why do we want to reset fail fast?  It's frustrating when a flaky test somewhere kills a bunch of other tests, and we don't know which one(s) would have failed.

Reply:
Multi nodes on macos and windows are failing because of the runner size.